lib/errors.ts

```diff
@@ -149,18 +150,12 @@ async function getSentry() {
 	return await import('@sentry/node');
 }
 
-export async function handleError(error: any) {
+export async function handleError(error: Error) {
 	// Set appropriate exitCode
 	process.exitCode =
-		error.exitCode === 0
+		(error as BalenaError).exitCode === 0
 			? 0
-			: parseInt(error.exitCode, 10) || process.exitCode || 1;
-
-	// Handle non-Error objects (probably strings)
-	if (!(error instanceof Error)) {
-		ErrorsModule.printErrorMessage(String(error));
-		return;
-	}
+			: Math.trunc((error as BalenaError).exitCode) || process.exitCode || 1;
```

Comment:
Fractional exit code? `exitCode === 3.1415` ? 😁  
(Jocose review comment... not a request for changes!)